server/services/curiousService.test.ts

```diff
@@ -69,19 +71,26 @@ describe('curiousService', () => {
       expect(curiousClient.getLearnerNeurodivergence).toHaveBeenCalledWith(prisonNumber, systemToken)
     })
 
-    it('should handle retrieval of prisoner support needs given Curious returns an unexpected error', async () => {
+    it('should handle retrieval of prisoner support needs given Curious returns an unexpected error for the learner profile', async () => {
       // Given
       const prisonNumber = 'A1234BC'
       const username = 'a-dps-user'
+
       const systemToken = 'a-system-token'
       hmppsAuthClient.getSystemClientToken.mockImplementation(() => Promise.resolve(systemToken))
 
+      const curiousApiError = {
+        message: 'Internal Server Error',
+        status: 500,
```

Comment:
Am I right in saying the message `Retrieving data from Curious is currently unavailable. Please try again later` is displayed in this scenario (and for any 4xx/5xx error, except for a 404)?